internal/generator/generator.go

```diff
@@ -123,8 +123,12 @@ func GetOptionSpec(filePath, optionsStructName, tagName string) (*OptionSpec, []
 	var warnings []string
 	for idx := range fields {
 		field := fields[idx]
+		fieldName := normalizeName(types.ExprString(field.Type))
+
+		if len(field.Names) > 0 {
```

Comment:
In some other cases we will always calculate the line `126`. It can be more effective when we do like 

```
var fieldName string
if len() >0{ fieldName = "foo" }else{ fieldName = "bar" }
```